cmd/zetae2e/local/local.go

```diff
@@ -310,6 +310,7 @@ func localE2ETest(cmd *cobra.Command, _ []string) {
 		if !skipPrecompiles {
 			precompiledContractTests = []string{
 				e2etests.TestZetaPrecompilesPrototypeName,
+				e2etests.TestZetaPrecompilesStakingName,
```

Comment:
Was mistakenly introduce in previous PR but can we remove these Zeta prefixes? It looks making the name long unnecessarily